autonomy/deploy/image.py

```diff
@@ -54,61 +39,25 @@ class ImageProfiles:  # pylint: disable=too-few-public-methods
     ALL = (CLUSTER, DEVELOPMENT, PRODUCTION)
 
 
-def build_image(  # pylint: disable=too-many-arguments
```

Comment:
Why not keep the version and push?

Reply:
@angrybayblade see version question is unresolved

Reply:
We don't want to push any images using the CLI tool that should not be the job for our tool and we can set versions using env vars https://github.com/valory-xyz/open-autonomy/pull/1221/files/9c29677b02c0b5c6a5e25c11da7191782f1fdab2#r953595612